docs/book/v4/migration/v3-to-v4.md

```diff
@@ -1,3 +1,29 @@
 # Migration from Version 3 to 4
 
 Version 4 of `laminas-stdlib` contains a number of backwards incompatible changes. This guide is intended to help you upgrade from the version 3 series to version 4.
+
```

Comment:
The page should have a structure similar to this:

- New Features
- Signature Changes
- Deprecations

Example: https://docs.laminas.dev/laminas-form/v3/migration/v2-to-v3/#migration-from-version-2-to-3